### PR TITLE
Calendar module, xiboIC update and fixes

### PR DESCRIPTION
**Pull request by maurofmferrao**
- Calendar module added
 - xiboIC added with globar vars/methods
 - Change input type input to text
 - Save widget that reloaded property form as layout fix
 - Header input
 - Color input: set default on onfocus

### CODE REVIEW

File: modules/templates/event-static.xml [hunk 1]
```diff
@@ -0,0 +1,1762 @@
...
+
+// Render
+$(target).find('img').xiboImageRender(properties);
+$(target).xiboLayoutScaler(properties);
+
+var runOnVisible = function() {
+    $(target).xiboTextRender(properties, items);
+};
+
+if (xiboIC.checkVisible()) {
```
Comment (dasgarner) [inline modules/templates/event-static.xml:109]: Does this need to be in `onVisible`?

File: modules/templates/article-static.xml [hunk 2]
```diff
@@ -138,18 +137,8 @@ if (properties.copyright) {
...
 
 // Render
-$(target).find('img').xiboImageRender(globalOptions);
-$(target).xiboLayoutScaler(globalOptions);
-
-var runOnVisible = function() {
-    $(target).xiboTextRender(properties, items);
-};
-
-if (xiboIC.checkVisible()) {
```
Comment (dasgarner) [inline modules/templates/article-static.xml:148]: Do we need an `onVisible` node to replace this?

File: ui/src/style/common.scss [hunk 3]
```diff
@@ -206,7 +206,7 @@ $fa-font-path: "~font-awesome/fonts";
     @extend .fa-calendar-o;
 }
 
-.module-icon-calendaradvanced {
+.module-icon-ics-calendar {
```
Comment (dasgarner) [inline ui/src/style/common.scss:209]: I suppose we need to think about how we do this!

**Review by dasgarner — APPROVED** [review]
I raised a few points to address in the next PR - you can merge this one I think!

File: modules/templates/article-static.xml (hunk 2 above)
Comment (maurofmferrao) [reply modules/templates/article-static.xml:148]: I forgot to update these files, I will update my branch ( and the PR ) with it

**Pull request merged**